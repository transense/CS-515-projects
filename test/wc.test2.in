j
za
yxp
rrdb
fwbhi
ertsio
vtlmqhj
nriyyvuv
zekjlxslu
z
im
vhf
acgw
dwgxn
kcwklu
pktynzo
htoktylt
xzbuwjfuh
d
qf
fgo
wvif
sgsww
xvhzaw
ozldqya
kzaxgibi
fzdowhbtz
v
qm
xxl
sbkl
gpkav
uwaqdc
dcapjvk
phrsuzvh
bbkkevprg
c
em
kae
zxia
nklnw
wmzrjb
jwrfxet
gpoymhhx
gwwrjnzro
n
al
jkg
bpff
hpumw
ulbyvz
fmbuhoy
xjjhfbcp
xvekjrgah
p
pn
jsp
llzt
bruyg
xqdpmm
izdrbae
eurrmqpa
pkphzwora
j
hh
ciz
hcbq
xcbko
gaudoj
abybbyy
xjnmcbgt
sxdlxzlgd
l
ro
moh
bqnf
mmrsz
hdjuup
dlameqm
axytvwqi
mkwxitcuq
y
no
kqo
jbuw
onlpb
lqbkxi
uwxfrbs
brjxyugt
mjdqteebd
t
je
fpq
lbyf
rynqi
occccn
jernlfm
ikvojvut
kxyaossfs
m
av
vqy
wgyy
xxpcm
vvopdk
quebnxd
wauzwaur
znrjlzhri
a bx bx bc io  110 
bl
shn
vyhi
zitgx
vdtyui
mwhswnn
fxdreiuj
ycggmioxj
g
tz
zht
ydbr
nvwvo
xdvbfi
npvquhi
cwpnirlk
gbygjtmzu
t
xe
jie
cejc
evbzd
gabumr
lnnzqnf
nohkbigj
gvnllptdr
