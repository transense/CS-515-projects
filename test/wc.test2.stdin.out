     135   140   828